stable/ibm-db2/ibm_cloud_pak/pak_extensions/common/db2u-install

```diff
@@ -13,7 +13,7 @@ storage_args=""
 db_name="BLUDB"
 helm_file=""
 advanced_install="false"
-arch="$(uname -p)"
+arch="amd64"
```

Comment:
The default value needs to be changed to `x86_64` since it is more generic.

Reply:
Hi @hrishikesh77, sure I can do that, I just figured i'd set it to 'amd64' since the helm charts `_architecture.tpl` converts `x86_64` to `amd64`.